doc/antora.yml

```
@@ -1,4 +1,4 @@
-name: guide-name    # a brief module name. This will be included in the guide url.
-title: Guide Title  # a brief module title. This will not show up on the website.
+name: hazelcast-terraform-cloud    # a brief module name. This will be included in the guide url.
+title: Deploying Hazelcast Clusters on Cloud using Terraform  # a brief module title. This will not show up on the website.
```

We use repository names as module names.
```suggestion
name: terraform-quickstarts
title: Deploying Hazelcast Clusters on Cloud using Terraform
```